packages/twenty-front/src/pages/settings/accounts/SettingsNewAccount.tsx

```diff
@@ -0,0 +1,24 @@
+import { SettingsHeaderContainer } from '@/settings/components/SettingsHeaderContainer';
+import { SettingsPageContainer } from '@/settings/components/SettingsPageContainer';
+import { IconSettings } from '@/ui/display/icon';
+import { SubMenuTopBarContainer } from '@/ui/layout/page/SubMenuTopBarContainer';
+import { Breadcrumb } from '@/ui/navigation/bread-crumb/components/Breadcrumb';
+
+export const SettingsNewAccount = () => {
+  return (
+    <SubMenuTopBarContainer Icon={IconSettings} title="Settings">
+      <SettingsPageContainer>
+        <SettingsHeaderContainer>
```

Comment:
`SettingsHeaderContainer` is not needed. We created it for when a button is needed besides the Breadcrumb, which is not the case for this page.

Reply:
Got it